pkg/istio/authorizationpolicy/controller.go

```diff
@@ -111,11 +113,16 @@ func (c *Controller) processEvent(fn cache.KeyFunc, obj interface{}) {
 
 // Run starts the main controller loop running sync at every poll interval.
 func (c *Controller) Run(stopCh <-chan struct{}) {
-	go c.resync(stopCh)
+	if c.standAloneMode {
```

Comment:
Can you help me understand `standAloneMode`? Is this equal to `authPolicyControllerOnly`?

Reply:
The way I thought about it
`standAloneMode` - a setting for the authorizationpolicy controller to indicate that it needs to use the `Run` method of the  `serviceIndexInformer`, `configStoreCache ` and `adIndexInformer`, in the case where the value is false, these methods are run by the controller that instantiates the authorizationpolicy controller.

`authPolicyControllerOnly` - this is a command line parameter mainly used the in the `main` function to indicate that we would like to run the authpolicy controller only.